packages/gatsby-plugin-now/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "gatsby-plugin-now",
+  "version": "1.2.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "./build.sh",
+    "test-integration-once": "jest"
```

```suggestion
    "test-integration-once": "jest --env node --verbose --runInBand"
```

Can we keep it consistent with the other packages?

---

`--runInBand` and `--env node` are not necessary.

```suggestion
    "test-integration-once": "jest --verbose"
```